src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2020-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "emitters/plugin/x64/jit_emitter.hpp"
+#include "emitters/snippets/x64/jit_kernel_emitter.hpp"
+#include "emitters/snippets/cpu_kernel_executor_table.hpp"
+#include <cpu/x64/brgemm/brgemm.hpp>
+
+namespace ov {
+namespace intel_cpu {
+class BrgemmKernelExecutor;
+#define GET_OFF_BRGEMM_ARGS(field) offsetof(BrgemmKernelExecutor::call_args, field)
+
+struct BrgemmKernelConfig : public snippets::KernelExecutorBase::GenericConfig {
+    friend BrgemmKernelExecutor;
+public:
+    BrgemmKernelConfig(const element::Type& in0_dtype, const element::Type& in1_dtype, float beta,
+                       bool is_with_amx, bool is_with_comp,
+                       size_t M = 0, size_t N = 0, size_t K = 0,
+                       size_t LDA = 0, size_t LDB = 0, size_t LDC = 0);
+    BrgemmKernelConfig() = default;
+    bool is_completed() const override;
+    size_t hash() const;
+    bool operator==(const BrgemmKernelConfig& rhs) const;
+    bool operator!=(const BrgemmKernelConfig& rhs) const;
+    std::string print() const;
```

Comment:
Fair enough. Fixed.
Also hid the methods under SNIPPETS_DEBUG_CAPS, because they are used only in src/plugins/intel_cpu/src/emitters/snippets/x64/verbose.cpp